### PR TITLE
feat: Tag Updates for additional colors

### PR DESCRIPTION
**Pull request by ccedrone**
Adding in min-width, success, and danger for Tag

### CODE REVIEW

**Comment by ccedrone** [comment]
https://github.com/cengage/react-magma/issues/398

**Review by orion-cengage — APPROVED** [review]
New tag colors and min-width look good.

**Comment by jamiesmyln** [comment]
The changes look good, it just needs a changeset before you merge.

File: .changeset/additional-colors.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"react-magma": patch
```
Comment (aaronmaturen) [inline .changeset/additional-colors.md:2]: react-magma-dom

**Pull request merged**